win32/mxe/settings.mk

```diff
@@ -8,6 +8,6 @@ MXE_TARGETS := i686-w64-mingw32.shared
 #SOURCEFORGE_MIRROR := downloads.sourceforge.net
 # The three lines below makes `make` build these "local
 # packages" instead of all packages.
-LOCAL_PKG_LIST := gettext sdl sdl2 boost portaudio ffmpeg portmidi pango gdk-pixbuf librsvg libsigc++ libxml++ glew libcdio opencv libepoxy
+LOCAL_PKG_LIST := gettext sdl sdl2 boost portaudio ffmpeg portmidi pango gdk-pixbuf librsvg libsigc++ libxml++ glew libcdio opencv libepoxy jsoncpp openssl cpp-netlib
```

Comment:
openssl is not needed as package since [it's included within cpp-netlib](https://github.com/mxe/mxe/blob/master/src/cpp-netlib.mk#L12) if i'm correct. Also libcdio can be removed since we don't use it anymore. Same goes for sdl (removed) since this one is refferenced within ffmpeg and we don't use sdl anymore.